lib/ts/index.ts

```diff
@@ -143,6 +143,15 @@ export default class AuthHttpRequest {
             userContext: getNormalisedUserContext(userContext)
         });
     };
+
+    static getAccessToken = async (input?: { userContext?: any }): Promise<string | undefined> => {
```

Comment:
this funtion shuold be in recipe interface and recipe implementation should have this code.

Reply:
Getting the token from storage is already customizable through `windowHandler`, and I think it'd be confusing:
- We are not using this function internally when getting the token
- We'd have to add an overrideable `setAccessToken` (and maybe others for refresh token, etc.)